src/cdk/a11y/key-manager/list-key-manager.spec.ts

```diff
@@ -759,7 +759,7 @@ describe('Key managers', () => {
       expect(keyManager.activeItemIndex)
           .toBe(0, `Expected first item of the list to be active.`);
 
-      keyManager.updateActiveItemIndex(1);
+      keyManager.setActiveItem(1);
```

Comment:
It looks like calling `setActiveItem` has a different effect than `updateActiveItemIndex` because it looks like this test is failing now

Reply:
ops, looks like I replaced it with the wrong method. Should be good to go now.